SFFeatureFlag/Classes/Utils/Extensions/String+Date.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  String+Date.swift
+//  SFFeatureFlag
+//
+//  Created by Maja Sapunova on 12.6.21.
+//
+
+import Foundation
+
+extension String {
+    
+    func toDate(format: String = "dd-MM-yyyy") -> Date? {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = format
+        dateFormatter.locale = Locale.current
```

Comment:
[optional]

You can just write .current

Reply:
Done